<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial_connection_widget_form</class>
 <widget class="QWidget" name="Serial_connection_widget_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="devices_combobox_"/>
   </item>
   <item>
    <widget class="QToolButton" name="rescan_button_">
     <property name="toolTip">
      <string>Rescan serial device</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="example/Serial_connection_widget_example.qrc">
       <normaloff>:/icons/rescan_icon</normaloff>:/icons/rescan_icon</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="baudrates_combobox_">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connect_button_">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>connect_button_</tabstop>
  <tabstop>devices_combobox_</tabstop>
  <tabstop>rescan_button_</tabstop>
 </tabstops>
 <resources>
  <include location="example/Serial_connection_widget_example.qrc"/>
 </resources>
 <connections/>
</ui>
